src/execution/execute.ts

```diff
@@ -1478,90 +1483,15 @@ export const defaultFieldResolver: GraphQLFieldResolver<unknown, unknown> =
  *
  * This function does not support incremental delivery (`@defer` and `@stream`).
  * If an operation which would defer or stream data is executed with this
- * function, each `InitialIncrementalExecutionResult` and
- * `SubsequentIncrementalExecutionResult` in the result stream will be replaced
- * with an `ExecutionResult` with a single error stating that defer/stream is
- * not supported.  Use `experimentalSubscribeIncrementally` if you want to
- * support incremental delivery.
+ * function, the result stream will be replaced with an `ExecutionResult`
+ * with a single error stating that defer/stream is not supported.
```

Comment:
I think that with the second commit, this needs to be tweaked a bit => as far as I can tell, field errors are thrown for each payload attempting to utilize incremental delivery, and so the returned `ExecutionResult` may contain data and potentially multiple errors, one of which being the aforementioned field error.

Reply:
Thanks for catching. This comment is updated now.